examples/kubernetes/static-provisioning/README.md

```diff
@@ -0,0 +1,59 @@
+# Static Provisioning
+
+## Context
+
+FSxZ resources can be mounted to a give pod either statically or dynamically.
```

Comment:
Not going to dive super deep into this one, but most of the comments left on the README for dynamic provisioning are also applicable here.